Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java

```diff
@@ -26,13 +26,15 @@ protected void onCreate(Bundle savedInstanceState) {
         ((App) getApplication()).getAppComponent().inject(this);
         activePlan = childPlanRepository.getActivePlan();
         if (activePlan != null) {
-            Intent taskList = new Intent(MainActivity.this, TaskListActivity.class);
-            startActivity(taskList);
-            setUpView();
+            Intent planList = new Intent(MainActivity.this, TaskListActivity.class);

```

Comment:
taskList was more precise. Stay with that, that was good idea.

Reply:
sure, rolled back by mistake, thanks